github_connector/report/github_repository_branch_rule_info_report.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2020: Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models, tools
+
+
+class GithubRepositoryBranchRuleInfoReport(models.Model):
+    _name = "github.repository.branch.rule.info.report"
+    _description = "Github Repository Branch Rule Info Report"
+    _auto = False
+
```

Comment:
Maybe you can use 'github.analysis.rule.mixin' here

Reply:
Interesting, it's possible to add and more re-use mixin function.